### ISSUE TITLE
File facet list and files summary disagree on file types

### ISSUE THREAD

**Issue by NoopDog** [DataBiosphere/azul#297]
When we display the file summaries when clicking the download button, we  are seeing there is a difference in the number of file types in the "file" facet terms and the file/summary api.

The file facets lists "unknown" but the summary does not and this causes an error on the client when we cant get the size for "unknown" file type.

Look at:

https://service.dev.explore.data.humancellatlas.org/repository/files/summary?filters=%7B%7D

vs

https://service.dev.explore.data.humancellatlas.org/repository/specimens?from=1&size=1&filters=%7B%7D

You will see that `unknown` is is the files facet terms but not in the summary.

```
{
  "fileCount": 852,
  "specimenCount": 1,
  "projectCount": 31,
  "totalFileSize": 2637013583,
  "organCount": 1,
  "donorCount": 1,
  "labCount": 1,
  "totalCellCount": 300000,
  "fileTypeSummaries": [
    {
      "fileType": "fastq.gz",
      "count": 852,
      "totalSize": 2637013583
    },
    {
      "fileType": "txt",
      "count": 409,
      "totalSize": 1695109696
    },
    {
      "fileType": "pdf",
      "count": 188,
      "totalSize": 1637753403
    },
    {
      "fileType": "bam",
      "count": 100,
      "totalSize": 1746790388
    },
    {
      "fileType": "results",
      "count": 94,
      "totalSize": 2277021986
    },
    {
      "fileType": "log",
      "count": 92,
      "totalSize": 1612285692
    },
    {
      "fileType": "cnt",
      "count": 68,
      "totalSize": 1606144620
    },
    {
      "fileType": "model",
      "count": 68,
      "totalSize": 1616910324
    },
    {
      "fileType": "theta",
      "count": 68,
      "totalSize": 1625549824
    },
    {
      "fileType": "time",
      "count": 68,
      "totalSize": 1606144404
    }
  ],
  "organSummaries": [
    {
      "organType": "brain",
      "countOfDocsWithOrganType": 852,
      "totalCellCountByOrgan": 300000
    }
  ]
}
```
VS

```
"fileFormat": {
      "terms": [
        {
          "term": "fastq.gz",
          "count": 36
        },
        {
          "term": "bam",
          "count": 23
        },
        {
          "term": "cnt",
          "count": 22
        },
        {
          "term": "log",
          "count": 22
        },
        {
          "term": "model",
          "count": 22
        },
        {
          "term": "pdf",
          "count": 22
        },
        {
          "term": "results",
          "count": 22
        },
        {
          "term": "theta",
          "count": 22
        },
        {
          "term": "time",
          "count": 22
        },
        {
          "term": "txt",
          "count": 22
        },
        {
          "term": "unknown",
          "count": 2
        }
      ],
      "total": 36,
      "type": "terms"
    }
```

For now we will make it so that we do not attempt to display information about file types that are not in the summary which will mask this error but we need to decide how to handle this.

Question:
Will the unknown type files appear in the download manifest?

Probably the back end can include the size of the `unknown` file types in the summary unless the size is unknown as well. 

Or we can not show the unknown files in the file facets list.

Or.. are we expecting not to see the unknown files in higher environments. 

Cheers,
Dave

┆Issue is synchronized with this [JIRA Story](https://ucsc-cgl.atlassian.net/browse/AZUL-197)
┆Project Name: azul
┆Issue Number: AZUL-197

**Comment by theathorn** [comment]
@achave11 says this should be fixed by #244.

**Comment by theathorn** [comment]
@mikebaumann @GPelayo It appear we are waiting for @hannes-ucsc to return before attempting a deploy to the HCA dev environment.
